tests/index.spec.ts

```diff
@@ -0,0 +1,25 @@
+import * as dotenv from 'dotenv';
+dotenv.config({ path: './test.env' });
+import { CreateUserCommand } from '../src/event-bus/create-user-command';
+import { createUser } from '../src/event-bus/create-user-handler';
+import { session } from '../src/neo4j';
+import { expect } from 'chai';
+
+describe('Test createUserHandler', () => {
+  it('should return a user', async () => {
```

Comment:
a descrição do `it` não está muito boa, ele não deveria retornar um usuário, deveria apenas criar